spec/intl.html

```diff
@@ -1578,55 +1272,6 @@ <h1>Intl.DateTimeFormat.prototype.formatRangeToParts ( _startDate_, _endDate_ )<
         1. Return ? FormatDateTimeRangeToParts(_dtf_, _x_, _y_).
       </emu-alg>
     </emu-clause>
-
-    <emu-clause id="sec-intl.datetimeformat.prototype.resolvedoptions">
-      <h1>Intl.DateTimeFormat.prototype.resolvedOptions ( )</h1>
-
-      <p>This function provides access to the locale and options computed during initialization of the object.</p>
-
-      <emu-alg>
-        1. Let _dtf_ be the *this* value.
-        1. If the implementation supports the normative optional constructor mode of <emu-xref href="#legacy-constructor"></emu-xref>, then
-          1. Set _dtf_ to ? UnwrapDateTimeFormat(_dtf_).
-        1. Perform ? RequireInternalSlot(_dtf_, [[InitializedDateTimeFormat]]).
-        1. <ins>Let _format_ be _dtf_.[[DateTimeFormat]].</ins>
-        1. Let _options_ be OrdinaryObjectCreate(%Object.prototype%).
-        1. For each row of <emu-xref href="#table-datetimeformat-resolvedoptions-properties"></emu-xref>, except the header row, in table order, do
-          1. Let _p_ be the Property value of the current row.
-          1. If there is an Internal Slot value in the current row, then
-            1. <del>Let _v_ be the value of _dtf_'s internal slot whose name is the Internal Slot value of the current row.</del>
-            1. <ins>If _p_ is neither *"hourCycle"* nor *"hour12"*, or if _format_ has a field [[hour]], then</ins>
-              1. <ins>Let _v_ be the value of _dtf_'s internal slot whose name is the Internal Slot value of the current row.</ins>
-            1. <ins>Else,</ins>
-              1. <ins>Let _v_ be *undefined*.</ins>
```

Comment:
I took another look at this today, with an eye to merging it. But I noticed that this change hasn't been incorporated upstream yet: https://tc39.es/ecma402/#sec-intl.datetimeformat.prototype.resolvedoptions
Could you take another look at it?

Reply:
This case is handled in [CreateDateTimeFormat](https://tc39.es/ecma402/#sec-createdatetimeformat), step 44:
> 44. If bestFormat has a field [[hour]], then
>        a. Set dateTimeFormat.[[HourCycle]] to hc.

Which means the Temporal change to remove that step needs to be removed.

Reply:
Got it. Thanks!